bench/src/main/scala/org/http4s/bench/CirceJsonStreamBench.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package bench
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import fs2.Chunk
+import fs2.Stream
+import io.circe._
+import org.http4s.circe._
+import org.openjdk.jmh.annotations._
+
+import java.util.concurrent.TimeUnit
+
+// sbt "bench/jmh:run -i 10 -wi 10 -f 2 -t 1 org.http4s.bench.CirceJsonStreamBench"
+@BenchmarkMode(Array(Mode.AverageTime))
```

Comment:
Most benchmarks often use a "Throughput" mode, rather than average. Perhaps that would help smooth out results error?